File: dehaze-android/sdk/src/main/java/com/pei/dehaze/sdk/model/user/UserForm.java
package com.pei.dehaze.sdk.model.user;

import com.pei.dehaze.sdk.model.EnableStatus;
import lombok.Data;
import java.util.List;

/**
 * 用户表单类型模型类
 */
@Data
public class UserForm {
    /**
     * 用户头像
     */
    private String avatar;
    
    /**
     * 部门ID
     */
    private Integer deptId;
    
    /**
     * 邮箱
     */
    private String email;
    
    /**
     * 性别
     */
    private Gender gender;

    /**
     * 用户ID
     */
    private Integer id;

    private String mobile;

    /**
     * 昵称
     */
    private String nickname;

    /**
     * 角色ID集合
     */
    private List<Integer> roleIds;

    /**
     * 用户状态
     */
    private EnableStatus status;
    
    /**
     * 用户名
     */
    private String username;
}

File: dehaze-go/model/bo/user_form.go
package bo

// UserFormBO 用户表单业务对象
type UserFormBO struct {
	// 用户ID
	ID int64 `json:"id"`
	// 用户名
	Username string `json:"username"`
	// 用户昵称
	Nickname string `json:"nickname"`
	// 手机号
	Mobile string `json:"mobile"`
	// 性别(1:男;2:女)
	Gender int8 `json:"gender"` // 修正类型为int8以匹配Java的Integer类型
	// 用户头像
	Avatar string `json:"avatar"`
	// 用户邮箱
	Email string `json:"email"`
	// 状态(1:启用;0:禁用)
	Status int8 `json:"status"` // 修正类型为int8以匹配Java的Integer类型
	// 部门ID
	DeptID int64 `json:"deptId"` // 修正类型为int64以匹配Java的Long类型
	// 角色ID集合
	RoleIds []int64 `json:"roleIds"` // 修正类型为int64以匹配Java的Long类型
}
